src/data/automation_i18n.ts

```diff
@@ -20,6 +20,8 @@ import {
 import { EntityRegistryEntry } from "./entity_registry";
 import { FrontendLocaleData } from "./translation";
 
+export type Weekday = "sun" | "mon" | "tue" | "wed" | "thu" | "fri" | "sat";
```

Comment:
This should be in the typing of the `TimeCondition` <https://github.com/home-assistant/frontend/blob/dev/src/data/automation.ts#L245>